src/Sylius/Bundle/PayumBundle/Resources/config/services/provider.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
+
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="sylius.payum_provider.payment_description" class="Sylius\Bundle\PayumBundle\Provider\PaymentDescriptionProvider">
```

Comment:
It's not strictly Payum connected, so I think it can be called `sylius.payment_description_provider`.